apis/kueue/webhooks/workload_webhook_test.go

```diff
@@ -301,7 +301,6 @@ func TestValidateWorkload(t *testing.T) {
 				Admit(testingutil.MakeAdmission("cluster-queue", "@invalid").Obj()).
 				Obj(),
 			wantErr: field.ErrorList{
-				field.Invalid(specField.Child("admission", "podSetFlavors").Index(0).Child("name"), nil, ""),
```

Comment:
what changed in the code that made this test change?

Reply:
This test change has nothing to do with my feature addition. However, as mentioned in https://github.com/kubernetes-sigs/kueue/pull/362#discussion_r957749549, the bug in the test code was missed because pkg apis/kueue/webhook was not tested.

You can reproduce it in the main branch with `go test ./apis/kueue/webhooks/...` command.

Reply:
Yes, exactly...